src/providers/dataProvider/index.ts

```diff
@@ -216,7 +216,7 @@ const auditForUpdatedChanges = async (
     activityDetail: `Previous values: ${JSON.stringify(difference)}`,
     resource: constants.R_ITEMS,
     index: record.id as number,
```

Comment:
@TahaKhanAbdalli  - I don't see why we are using `previousData` in the next line.  Are the two ids always the same?

Reply:
Yes the `ids` are same, when updating `react-admin` provides the full current document to update in `record.data` but when we are manually updating in the test we provided `partial` of the data to update which was causing issues in testing. Or I can remove this and provide the id when updating the document then we can use `record.data.id`

Reply:
Aah, I see.  That's ok :-)